docs/projects/tad/reporting-standard/latest.md

```diff
@@ -62,10 +62,21 @@ the examples in the next section.
 A `system_card` contains the following information.
 
 1. `schema_version` (REQUIRED, string). Version of the schema used, for example "0.1a2".
-2. `name` (OPTIONAL, string). Name used to describe the system.
-3. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+2. `provenance` (OPTIONAL). In case this System Card is generated from another source file, this field
+can capture the historical context of the contents of this System Card.
+
+    1. `modifier` (OPTIONAL, string). Git-hash of the file containing the transformations that where used
```

Comment:
Reading this more closely, I don't think we mean a "git-hash of the file", but rather a "git commit hash"? And if that's the only type of value we support, shouldn't we just call it that?

Reply:
Added your suggested change